### ISSUE TITLE
Jupyter sandbox: user exception

### ISSUE THREAD

**Issue by ondrejhlavacek** [keboola/provisioning-bundle#67]
```
Sep 18 09:09:56  Loading tagged files from "["client_secrets.json","credentials.dat"]"
Sep 18 09:09:56  [E 07:09:56.125 NotebookApp] Exception while loading config file /etc/jupyter/jupyter_notebook_config.py
Sep 18 09:09:56      Traceback (most recent call last):
Sep 18 09:09:56        File "/opt/conda/lib/python3.6/site-packages/traitlets/config/application.py", line 562, in _load_config_files
Sep 18 09:09:56          config = loader.load_config()
Sep 18 09:09:56        File "/opt/conda/lib/python3.6/site-packages/traitlets/config/loader.py", line 457, in load_config
Sep 18 09:09:56          self._read_file_as_dict()
Sep 18 09:09:56        File "/opt/conda/lib/python3.6/site-packages/traitlets/config/loader.py", line 489, in _read_file_as_dict
Sep 18 09:09:56          py3compat.execfile(conf_filename, namespace)
Sep 18 09:09:56        File "/opt/conda/lib/python3.6/site-packages/ipython_genutils/py3compat.py", line 198, in execfile
Sep 18 09:09:56          exec(compiler(f.read(), fname, 'exec'), glob, loc)
Sep 18 09:09:56        File "/etc/jupyter/jupyter_notebook_config.py", line 93, in <module>
Sep 18 09:09:56          app.prepare_tagged_files(cfg, tags)
Sep 18 09:09:56        File "/opt/conda/lib/python3.6/site-packages/kbc_transformation/transformation.py", line 89, in prepare_tagged_files
Sep 18 09:09:56          raise ValueError("No files were found for tag: " + tag)
Sep 18 09:09:56      ValueError: No files were found for tag: client_secrets.json
```
throws

`Failed to create jupyter for '67432' kbc-us-east-1-docker-sandboxes-03 (Essential container in task exited).`

**Comment by ondrejhlavacek** [comment]
Related to https://github.com/keboola/provisioning-bundle/issues/41

**Comment by odinuv** [comment]
fixed in 7c86ed34c87761fa1ca55d82984d1e6dddc7dc40
